Tests/AppTests/Common/Helpers/Controllers/Registration/UserSessionHelper.swift

```diff
@@ -0,0 +1,20 @@
+import Vapor
+import Fluent
+@testable import App
+
+final class UserSessionHelper {
+ 
+  static func store() throws {
+    
+    for _ in 1...Int.random(min: 2, max: 10) {
+      
+      let user = User()
+      try user.save()
+      
+      let session = Session(userId: user.id!)
+      try session.save()
+      
```

Comment:
Remove this odd whitespaces at the end of scope )

Reply:
fixed!)